test-case-reporting/src/db.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('dotenv').config();
```

Comment:
Nit: typically it's a good idea to instead put the dotenv initialization in any of the entry-points. Here, we'd want it in `setup-db.ts` as well as `index.ts`.

Reply:
You can think about it like "the larger programs using this `db` module are in charge of making sure the environment is set up; I'm just responsible for using it."

Reply:
Done.